### ISSUE TITLE
OpenApi v3: Code assist for Link#operationRef

### ISSUE THREAD

**Issue by tfesenko** [RepreZen/KaiZen-OpenAPI-Editor#305]
Extracted from #289 
[Link#operationRef](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#link-object) is
> a relative or absolute reference to an OAS operation. This field is mutually exclusive with the operationId field, and must point to the fragment of a valid OAS definition.

For now, only show a relative operationRef:
```yaml
components:
  links:
    UserRepositories:
      # returns array of '#/components/schemas/repository'
      operationRef: '#paths~12.0~1repositories~1{$response.body#/username}'
```

**Comment by ghillairet** [comment]
If I take the default example, is this a valid operationRef?

```yaml
components:
  links:
    UserCommitHistory:
      operationId: listPets
      operationRef: "#/paths/~1pets/get/"
```

**Comment by andylowry** [comment]
@tfesenko @ghillairet I noticed that the editor understands pure JSON pointers ("#...") but not any other form of JSON ref (i.e. anything before the '#'). My interpretation of the specification is that full-blown JSON references should be handled as well. If that sounds right to you, should that be a separate issue?

**Comment by ghillairet** [comment]
@tfesenko Can be close.

Content assist for operationRef has been merged done in #332.

@andylowry External references should be available when you press Ctrl-Space. 

Remaining issues:

- Navigation does not work (open in https://github.com/RepreZen/KaiZen-OpenAPI-Editor/issues/335)
- Validation does not work when operationRef references an external operation (absolute ref) (open in  https://github.com/RepreZen/KaiZen-OpenAPI-Editor/issues/306).
